### DISCUSSION TITLE
Systemd Service Not Working Right?

### DISCUSSION THREAD

**Discussion by nickian** [coder/code-server#2313] [Q&A]
Can someone explain how the systemd service works for code-server? I'm very confused because I ran the install script, then this:

`sudo systemctl enable --now code-server@$USER`

I am the root user and I'm the only one using this machine, so I want the service to be system-wide. 

What I am confused about is that I can run `systemctl stop code-server@$USER` and the service says it is stopped. However, code-server is not stopped. It's still running and accessible. So it doesn't seem to be related to the actual app.

How can I ensure that the systemd service is actually controlling code-server? Also, can set this up so that it's not `@$USER`? I just want to be able to type `service cod-server restart`, etc.

Thanks!

**Comment by code-asher** [comment]
I think you should be able to just `systemctl enable --now code-server`.

I'm not sure why it wouldn't actually stop when using $USER though. To
be honest I haven't ever used this syntax before, I always enable things
globally or use `systemctl --user`.
